apps/src/applab/designElements/screen.jsx

```diff
@@ -198,47 +196,35 @@ export default {
     element.style.zIndex = 0;
     element.setAttribute('tabIndex', '1');
 
-    if (experiments.isEnabled('applabThemes')) {
-      if (!element.getAttribute('data-theme')) {
-        element.setAttribute(
-          'data-theme',
-          applabConstants.themeOptions[applabConstants.CLASSIC_THEME_INDEX]
-        );
-      }
+    if (!element.getAttribute('data-theme')) {
+      element.setAttribute(
+        'data-theme',
+        applabConstants.themeOptions[applabConstants.CLASSIC_THEME_INDEX]
+      );
+    }
 
-      if (element.style.backgroundColor === '') {
-        element.style.backgroundColor = this.themeValues.backgroundColor[
-          applabConstants.themeOptions[applabConstants.CLASSIC_THEME_INDEX]
-        ];
-      }
+    if (element.style.backgroundColor === '') {
+      element.style.backgroundColor = this.themeValues.backgroundColor[
+        applabConstants.themeOptions[applabConstants.CLASSIC_THEME_INDEX]
+      ];
     }
   },
   readProperty: function(element, name) {
     switch (name) {
       case 'theme':
-        if (experiments.isEnabled('applabThemes')) {
-          return element.getAttribute('data-theme');
-        } else {
-          throw `unknown property name ${name}`;
-        }
+        return element.getAttribute('data-theme');
       default:
         throw `unknown property name ${name}`;
     }
   },
   onPropertyChange: function(element, name, value) {
-    if (experiments.isEnabled('applabThemes')) {
-      switch (name) {
-        case 'theme': {
-          const prevValue =
-            element.getAttribute('data-theme') ||
-            applabConstants.themeOptions[applabConstants.CLASSIC_THEME_INDEX];
-          element.setAttribute('data-theme', value);
-          designMode.changeThemeForCurrentScreen(prevValue, value);
-          return true;
-        }
-        default:
-          break;
+    switch (name) {
```

Comment:
This should be an if statement instead of a switch statement (same for readProperty above)